javascript/ecmascript-2015/array-updates/searching-array-methods.md

```diff
@@ -116,8 +107,16 @@ ints.???(???)
 
 Fill the array with the first number divisible by `2` and `3`:
 
-```javacript
-let ints = ['2','5','4','5','6','5','8'];
+```javascript
+let ints = [
+  "2",
+  "5",
+  "4",
+  "5",
+  "6",
+  "5",
+  "8"
+];
 ints.???(ints.???(???))
```

Comment:
For the questions, we should remove `fill` as a hidden answer (since it won't ever b mentioned in the insight itself).

This is my suggestion:

```md
---
## Practice

Find the first number divisible by `4`:

```javascript
let ints = ["6", "2", "5", "6", "5", "8"];
ints.???(???);
```

* `find`
* `function (e) {return e % 4 === 0}`
* `includes`
* `function (e) {return e % 4}`
* `filter`
* `function (e) {return e === 4}`

---
## Revision

Fill the array with the first number divisible by `2` and `3`:

```javascript
let ints = [
  "2",
  "5",
  "4",
  "5",
  "6",
  "5",
  "8"
];
ints.fill(ints.???(???))
```

* `find`
* `e => e % 2 === 0 && e % 3 === 0`
* `findIndex`
* `includes`
* `e => e / 2 === 0 && e / 3 === 0`

```

Reply:
changed the PQ and RQ according to the suggestion